packages/engine/paima-funnel/src/cde/cardanoTransfer.ts

```diff
@@ -24,10 +25,18 @@ export default async function getCdeData(
 ): Promise<CdeCardanoTransferDatum[]> {
   let result = [] as CdeCardanoTransferDatum[];
 
+  const relationFilter =
+    RelationFilterType.Output |
+    RelationFilterType.Input |
+    RelationFilterType.StakeRegistration |
+    RelationFilterType.StakeDeregistration |
+    RelationFilterType.StakeDelegation |
+    RelationFilterType.Withdrawal;
```

Comment:
Hmm, this way is possible I guess, but we'd have to be clear in the docs that users have to manually parse the transaction to know if the address appears in the inputs or outputs (and if it appears in neither the inputs or the outputs, it's because it's one of the other types)

Reply:
I can't think of a particular nice way to do this other than to include the tx cbor as well (as mentioned originally here: https://github.com/PaimaStudios/paima-engine/issues/262)

Reply:
Also I just realized we also probably want `RequiredSigner` as an input type